### PR TITLE
Add sanity_single marker for single-cluster OCP sanity tests and remove kafka tests

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -7,6 +7,7 @@ markers =
     # Test types
     smoke: Mark tests as smoke tests.
     smoke_multi: Mark multi-cluster tests as smoke tests.
+    smoke_single: Mark single-cluster tests as smoke tests.
```
Comment (myakove) [inline pytest.ini:10]: I think it should be `smoke_single_cluster`, also for multi.

**Review by myakove — CHANGES_REQUESTED** [review]

File: pytest.ini (hunk 1 above)
Comment (rnetser) [reply pytest.ini]: done

**Review by myakove — APPROVED** [review]

**Pull request merged**